cmd/minikube/cmd/stop.go

```diff
@@ -81,6 +84,18 @@ func runStop(cmd *cobra.Command, args []string) {
 		profilesToStop = append(profilesToStop, cname)
 	}
 
+	if scheduledStop != "" {
```

Comment:
we should check if windows, then exit with usge that is not implemented on windows yet

Reply:
done